static/js/components/PersonalTab.js

```diff
@@ -25,19 +26,27 @@ class PersonalTab extends React.Component {
   updateGender(gender) {
     const { dispatch, profile: { profileCopy } } = this.props;
```

Comment:
Yes that's correct. It could probably use a better name, maybe `profileInEdit` or something like that